syft/frameworks/torch/hook.py

```diff
@@ -90,12 +87,12 @@ def __init__(
         syft.torch = TorchAttributes(torch, self)
 
         if self.local_worker is None:
-            """
-            Every TorchHook instance should have a local worker which is responsible for
```

Comment:
This should be formatting as a string comment since it is multiple lines

Reply:
I am a bit confused, the changes I commited have these as "#comment" rather than """strings.

Reply:
What I was trying to say was that the """ string was the correct way to format it for consistency

Reply:
Got it, However google style doc shows multiline comments using #.
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#385-block-and-inline-comments